src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -174,8 +174,8 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
                     </div>
                     <div>
                         <EditorFooter
-                            displayHotKeys={true}
```

Comment:
why are we replacing this with lockedTags/

Reply:
because `displayHotKeys` is not longer needed and `lockedTags` now is